### ISSUE TITLE
rangeIncomplete fires after clearing BOTH time pickers

### ISSUE THREAD

**Issue by jkallay1** [jonthornton/Datepair.js#96]
Starting with a time set in both time pickers, the event fires after clearing one and then again after clearing the other. This is semantically incorrect, as having an entirely empty range is different from having one with only one value.

**Comment by jonthornton** [comment]
Can you provide a use case for this? Is it causing problems as you build your app?

**Comment by jkallay1** [comment]
Yes. I'm using the rangeIncomplete event to mark the control as invalid and the rangeSelected event to mark it as valid for client-side validation purposes. This bug requires a special check for the case where both fields are empty in the rangeIncomplete handler.
